syntax = "proto3";
package DomainServices.OfferService;
option csharp_namespace = "DomainServices.OfferService.Contracts";

message OfferIdRequest {
	int32 OfferId = 1;
}